test/jdk/java/awt/color/ICC_Profile/MTGetData.java

```diff
@@ -102,6 +103,8 @@ private static void test(byte[] all, byte[] data1, byte[] data2, int tag)
                     icc.setData(tag, data1);
                     icc.setData(tag, data2);
                 }
+            } catch (CMMException cmmEx) {
```

Comment:
Don't we need to throw the IllegalArgumentException instead of CMMException here?